sklearn/feature_extraction/text.py

```diff
@@ -599,7 +599,7 @@ class HashingVectorizer(TransformerMixin, _VectorizerMixin, BaseEstimator):
         will be removed from the resulting tokens.
         Only applies if ``analyzer == 'word'``.
 
-    token_pattern : string
+    token_pattern : string, default=r"(?u)\\\b\\\w\\w+\\\b"
```

Comment:
We only need two slashes? (Like your original version)

```suggestion
    token_pattern : string, default=r"(?u)\\b\\w\\w+\\b"
```

Reply:
Right. Sorry.